blocks/media-upload-button/index.js

```diff
@@ -78,11 +92,19 @@ class MediaUploadButton extends Component {
 		}
 
 		if ( gallery ) {
+			const ids = this.props.value || [];
+			const attachments = getAttachmentsCollection( ids );
+			const currentState = ( this.props.value ) ? 'gallery-edit' : 'gallery';
```

Comment:
Should we do the same for a single media? I mean do we have something like 'edit'  and 'select' states for single media selector?

Reply:
Hi, @youknowriad we have edit states for single media like audio and video, but this states open with a modal to change details like sizes, captions etc of the item that we can now change in the Gutenberg block. So for single media I don't think we should use this states.